server/bin/pbench-server-put-shim.py

```diff
@@ -46,18 +45,22 @@ def main(cfg_name: str) -> int:
     result_string = (
         f"{config.TS} Status: Total no. of tarballs {counts.ntotal},"
         f" Successfully moved {counts.ntbs}, Encountered"
-        f" {counts.ntotal-counts.ntbs} failures"
+        f" {counts.ntotal-counts.ntbs} failures and {counts.nerr} Errors"
```

Comment:
It's not going to be at all clear from this report what the difference is between "failures" and "errors", so I think this should be reworded. Or, if we can't come up with a good distinction between them, they should simply be added together.

Reply:
I think we can add them together.